src/legacy/ui/public/vis/editors/default/controls/sub_metric.tsx

```diff
@@ -64,6 +64,7 @@ function SubMetricParamEditor({
 
   return (
     <>
+      <EuiSpacer size="m" />
       <EuiFormLabel>{aggTitle}</EuiFormLabel>
       <EuiSpacer size="s" />
       <DefaultEditorAggParams
```

Comment:
If we wrap this into `<EuiFormRow display="rowCompressed" fullWidth label={aggTitle}>`, we won't need spacers and additional styles in `_agg.scss`.